lib/rust/ensogl/component/text/src/font/family/src/lib.rs

```diff
@@ -67,11 +66,24 @@ pub use owned_ttf_parser::Width;
 /// eliminate accidental mistakes, the same way as it's done in CSS:
 /// https://stackoverflow.com/questions/17967371/are-property-values-in-css-case-sensitive
 #[allow(missing_docs)]
-#[derive(Clone, Debug, Deref, Display, Hash, PartialEq, Eq)]
+#[derive(Clone, Debug, Hash, PartialEq, Eq)]
 pub struct Name {
     pub normalized: String,
 }
 
+impl std::ops::Deref for Name {
+    type Target = str;
+    fn deref(&self) -> &Self::Target {
+        &self.normalized
+    }
+}
+
+impl std::fmt::Display for Name {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.write_str(&self.normalized)
+    }
+}
+
```

Comment:
`prelude` was handling this through `derive_more`. Would it hurt compile time for this crate to use `derive_more` directly?

Reply:
It doesn't impact it. `derive_more` is already compiled before anyway.